Common++/header/IpAddress.h

```diff
@@ -579,9 +540,9 @@ namespace pcpp
 		uint32_t m_NetworkPrefix;
 		uint32_t m_Mask;
 
-		bool isValidNetmask(const std::string& netmask);
+		bool isValidNetmask(const IPv4Address& netmaskAddress);
```

Comment:
it's still a useful private method, but we pass `IPv4Address` instead.

Reply:
Maybe we should modify `IPv4Network` / `IPv6Network` in a separate PR?

Reply:
In addition, if you check closely, not much code is affected by this change. The behavior of using `IPv4Network` / `IPv6Network` is still the same, and only the private methods are changed.

Reply:
got it, I probably overlooked it. I'll take a closer look soon